toolkit/tools/scheduler/schedulerutils/printresults.go

```diff
@@ -105,113 +110,119 @@ func PrintBuildSummary(pkgGraph *pkggraph.PkgGraph, graphMutex *sync.RWMutex, bu
+		keys := sliceutils.MapToSlice(skippedSRPMsTests)
+		sort.Strings(keys)
+		for _, skippedSRPMsTest := range keys {
+			logger.Log.Infof("--> %s", filepath.Base(skippedSRPMsTest))
 		}
 	}
 
-	if len(failedSRPMs) != 0 {
-		logger.Log.Info("Failed SRPMs:")
-		for _, failure := range failedSRPMs {
-			logger.Log.Infof("--> %s , error: %s, for details see: %s", failure.Node.SRPMFileName(), failure.Err, failure.LogFile)
+	if len(builtSRPMs) != 0 {
+		logger.Log.Info(color.GreenString("Built SRPMs:"))
+		keys := sliceutils.MapToSlice(builtSRPMs)
+		sort.Strings(keys)
+		for _, builtSRPM := range keys {
+			logger.Log.Infof("--> %s ", filepath.Base(builtSRPM))
 		}
 	}
 
-	if len(failedSRPMsTests) != 0 {
-		logger.Log.Info("Failed SRPMs tests:")
-		for _, failure := range failedSRPMsTests {
-			logger.Log.Infof("--> %s , error: %s, for details see: %s", failure.Node.SRPMFileName(), failure.Err, failure.LogFile)
+	if len(testedSRPMs) != 0 {
+		logger.Log.Info(color.GreenString("Tested SRPMs:"))
+		keys := sliceutils.MapToSlice(testedSRPMs)
+		sort.Strings(keys)
+		for _, testedSRPM := range keys {
+			logger.Log.Infof("--> %s", filepath.Base(testedSRPM))
 		}
 	}
 
-	if len(blockedSRPMs) != 0 {
-		logger.Log.Info("Blocked SRPMs:")
-		for srpm := range blockedSRPMs {
-			logger.Log.Infof("--> %s", filepath.Base(srpm))
+	if len(unresolvedDependencies) != 0 {
+		logger.Log.Info(color.RedString("Unresolved dependencies:"))
+		keys := sliceutils.MapToSlice(unresolvedDependencies)
+		sort.Strings(keys)
+		for _, unresolvedDependency := range keys {
+			logger.Log.Infof("--> %s", filepath.Base(unresolvedDependency))
 		}
 	}
 
-	if len(blockedSRPMsTests) != 0 {
-		logger.Log.Info("Blocked SRPMs tests:")
-		for srpm := range blockedSRPMsTests {
-			logger.Log.Infof("--> %s", filepath.Base(srpm))
+	if len(blockedSRPMs) != 0 {
+		logger.Log.Info(color.RedString("Blocked SRPMs:"))
+		keys := sliceutils.MapToSlice(blockedSRPMs)
+		sort.Strings(keys)
+		for _, blockedSRPM := range keys {
+			logger.Log.Infof("--> %s", filepath.Base(blockedSRPM))
 		}
 	}
 
-	if len(unresolvedDependencies) != 0 {
-		logger.Log.Info("Unresolved dependencies:")
-		for dependency := range unresolvedDependencies {
-			logger.Log.Infof("--> %s", dependency)
+	if len(blockedSRPMsTests) != 0 {
+		logger.Log.Info(color.RedString("Blocked SRPMs tests:"))
+		keys := sliceutils.MapToSlice(blockedSRPMsTests)
+		sort.Strings(keys)
+		for _, blockedSRPMsTest := range keys {
+			logger.Log.Infof("--> %s", filepath.Base(blockedSRPMsTest))
 		}
 	}
 
 	if len(rpmConflicts) != 0 {
-		conflictsLogger("RPM conflicts with toolchain: ")
+		conflictsLogger(color.RedString("RPM conflicts with toolchain:"))
+		sort.Strings(rpmConflicts)
 		for _, conflict := range rpmConflicts {
 			conflictsLogger("--> %s", conflict)
 		}
 	}
 
 	if len(srpmConflicts) != 0 {
-		conflictsLogger("SRPM conflicts with toolchain: ")
+		conflictsLogger(color.RedString("SRPM conflicts with toolchain:"))
```

Comment:
The `conflictsLogger` is already set-up to print the output as an error or as an info log depending on the situation. For cases, where it's only an info (for instance where we allow toolchain rebuilds and then they are an expected occurrence), I think we shouldn't print the output in red. It would be sending the wrong message.

Reply:
Thanks for highlighting this. Let me color it green (if allowToolchainRebuilds=y) and red otherwise